### ISSUE TITLE
Question about windows decorations and a patch 

### ISSUE THREAD

**Issue by jfk88** [lxqt/pcmanfm-qt#1107]
I've applied this snapping patch to openbox: https://github.com/emilypeto/openbox-window-snap

The patch fully works for windows that are not pcmanfm. Otherwise, pcmanfm instances are snapping but without the decorations (or with the decorations going beyond the frame) and the bottom behind the taskbar: 

Do you know what would cause this behavior?

**Comment by tsujan** [comment]
1. That's not pcmanfm-qt but obconf-qt. pcmanfm-qt has no relation to openbox or any WM.
 2. You applied a patch; what do you expect?

**Comment by jfk88** [comment]
> 1. That's not pcmanfm-qt but obconf-qt. pcmanfm-qt has no relation to openbox or any WM.
> 2. You applied a patch; what do you expect?

Yeah sorry I edited that 3 secs before your reply. Let me rephrase: what would be the reason for some windows, including pcmanfm instances, to "over-resized"?

**Comment by tsujan** [comment]
> what would be the reason for some windows, including pcmanfm instances, to "over-resized"?

The extra patch or its incompatibility with an openbox setting (which, again, is the patch's fault).

Please also note that openbox isn't a part of LXQt. And even if it was, we wouldn't support others' patches.
